.gitlab-ci.yml

```diff
@@ -140,23 +140,6 @@ publish-onchain:
   tags:
     - linux-docker
 
-# configures aws for fast uploads/syncs
-.s3-before-script:                &s3-before-script
```

Comment:
why is this being removed?

Reply:
moved to docker parity/awscli @TriplEight ?

Reply:
it's [here](https://github.com/paritytech/scripts/blob/master/awscli/config) now